components/AccountSelector/index.jsx

```diff
@@ -185,12 +185,16 @@ const AccountSelector = () => {
                   display='flex'
                   flexDirection='column'
                   justifyContent='space-between'
-                  width={11}
-                  height={11}
+                  width='300px'
+                  height='300px'
                   borderRadius={2}
                   p={3}
                 >
                   <Title>Choose an account</Title>
+                  <Text m={0}>
```

Comment:
This is not a ledger-only screen

Reply:
@Schwartz10 Shouldn't we be distinguishing between accounts that are Ledger-based vs. Private Key vs. Seed Phrase?

Reply:
You can only use one wallet type at a time, if you selected ledger at the start of onboarding, you're using ledger the whole way through.